packages/igx-templates/IgniteUIForAngularTemplate.ts

```diff
@@ -121,7 +121,7 @@ export class IgniteUIForAngularTemplate implements Template {
 	}
 	public setExtraConfiguration(extraConfigKeys: {}) { }
 
-	protected getBaseVariables(name: string) {
+	protected getBaseVariables(name: string): { [key: string]: string } {
```

Comment:
This can be on a higher level (like the Template interface) if it's not typed there

Reply:
It's `protected` and not described in the interface. There are no prior inheritance levels where this can be applied, it's a small QoL life 😅. Then again, it's not called anywhere outside of the template itself, so it can be removed.